### PR TITLE
support the report value in the dml and dql request

### PR DESCRIPTION
**Pull request by SimFG**
issue: https://github.com/milvus-io/milvus/issues/30436

### CODE REVIEW

File: examples/hello_cost.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+result = hello_milvus.query(expr=expr, output_fields=["random", "embeddings"])
+print(f"query after delete by expr=`{expr}` -> result: {result}\n")
+
+
+###############################################################################
+# 7. drop collection
+# Finally, drop the hello_milvus collection
+print(fmt.format(f"Drop collection `{collection_name}`"))
+utility.drop_collection(collection_name")
```
Comment (czs007) [inline examples/hello_cost.py:187]: Incorrect , an extra quotation mark was added.

File: pymilvus/client/abstract.py [hunk 2]
```diff
@@ -497,7 +513,7 @@ def __iter__(self) -> SequenceIterator:
 
     def __str__(self) -> str:
         """Only print at most 10 query results"""
-        return str(list(map(str, self[:10])))
+        return f"part data: {list(map(str, self[:10]))}, cost: {self.cost}"
```
Comment (czs007) [inline pymilvus/client/abstract.py:516]: data: ["{'id': 2, 'vector': [0.1950739, 0.80361205, 0.17314962, 0.074133284, 0.85392755, 0.8094358, 0.037969112, 0.2732632], 'g': 100}"]..., extra_info: {'cost': 21}

File: pymilvus/orm/mutation.py [hunk 3]
```diff
@@ -53,6 +53,10 @@ def succ_index(self):
     def err_index(self):
         return self._mr.err_index if self._mr else []
 
+    @property
+    def cost(self):
+        return self._mr.cost if self._mr else "0"
```
Comment (czs007) [inline pymilvus/orm/mutation.py:58]: change to int?

File: pymilvus/client/abstract.py [hunk 4]
```diff
@@ -234,10 +235,15 @@ def succ_index(self):
     def err_index(self):
         return self._err_index
 
+    @property
+    def cost(self):
```
Comment (XuanYang-cn) [inline pymilvus/client/abstract.py:239]: Please add comment to explain what does cost mean and what's the unit of cost. Is it time? is it memory related? or both? or other meanings?

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: pymilvus/client/abstract.py (hunk 4 above)
Comment (SimFG) [reply pymilvus/client/abstract.py:239]: is vcu

**Comment by czs007** [comment]
@SimFG remember pick to branch 2.4

**Review by XuanYang-cn — APPROVED** [review]

**Pull request closed without merging**